source/clear-linux/guides/maintenance/enable-user-space.rst

```diff
@@ -6,22 +6,24 @@ Create and enable a new user space
 This section provides steps to complete the following basic setup tasks for
 a newly installed |CL-ATTR| system:
 
-* Create a new user.
-* Update the OS to its most current version using `swupd`.
-* Install the most common applications for system administrators and
-  developers using bundles.
-* Set up a new user and add the new user to the `wheel` group.
-* Install a GUI to test `sudo` privileges.
+.. contents::
+   :local:
+   :depth: 1
 
-.. note::
-   Log in as the root user to complete the tasks in this
-   section.
+Prerequisites
+*************
+
+Log in as the root user to complete the tasks in this section. Enter:
```

Comment:
@mvincerx , you've made the assumption that the user has installed the desktop but if they didn't, they would be staring at the login: console prompt.  At that point they would need to type in "root" as the user and go through the steps of setting up the initial password for root.  If they did install the desktop then they would need to open a command window and then run the sudo -i command to set the root password.  What we could do is have them go to the alternate terminal using ctrl-alt-F2 where they would be looking at the login: console prompt.  Does that make sense?

Reply:
I addressed this possibility in the note in the new section "Recommended options."